extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/model/tasks/TaskSeries.java

```diff
@@ -15,57 +15,61 @@
  */
 package org.dataportabilityproject.transfer.rememberthemilk.model.tasks;
 
+import com.fasterxml.jackson.xml.annotate.JacksonXmlProperty;
 import com.google.api.client.util.Joiner;
-import com.google.api.client.util.Key;
 
+import java.util.ArrayList;
 import java.util.List;
 
 /** A tasks series, see: https://www.rememberthemilk.com/services/api/tasks.rtm */
 public class TaskSeries {
 
-  @Key("@id")
+  @JacksonXmlProperty(isAttribute = true, localName = "id")
   public int id;
 
-  @Key("@created")
+  @JacksonXmlProperty(isAttribute = true, localName = "created")
   public String created;
 
-  @Key("@modified")
+  @JacksonXmlProperty(isAttribute = true, localName = "modified")
   public String modified;
 
-  @Key("@name")
+  @JacksonXmlProperty(isAttribute = true, localName = "name")
   public String name;
 
-  @Key("@source")
+  @JacksonXmlProperty(isAttribute = true, localName = "source")
   public String source;
 
-  @Key("@authUrl")
-  public String url;
-
-  @Key("@location_id")
+  @JacksonXmlProperty(isAttribute = true, localName = "location_id")
   public String location_id;
 
-  @Key("tags")
+  @JacksonXmlProperty(localName = "url")
+  public String url;
+
+  @JacksonXmlProperty(localName = "tags")
   public String tags;
 
-  @Key("participants")
+  @JacksonXmlProperty(localName = "participants")
   public String participants;
 
-  @Key("notes")
+  @JacksonXmlProperty(localName = "notes")
   public Notes notes;
 
-  @Key("task")
-  public List<Task> tasks;
+  @JacksonXmlProperty(localName = "tasks")
+  public List<Task> tasks = new ArrayList<>();
+
+  public void setTask(Task singleTask) {
+    this.tasks.add(singleTask);
+  }
 
   @Override
   public String toString() {
     return String.format(
-        "TaskSeries(id=%d created=%s modified=%s name=%s source=%s authUrl=%s, notes=%s tasks:%s)",
+        "TaskSeries(id=%d, created=%s, modified=%s, name=%s, source=%s, notes=%s, tasks:%s)",
```

Comment:
Can we use this guava utility method in order to be less error prone https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Objects.ToStringHelper.html

Reply:
Done! IntelliJ has an option to auto-insert/generate this as well - made it super easy :)